src/mqueryfront/src/query/QueryMatches.js

```diff
@@ -85,7 +84,7 @@ const QueryMatches = (props) => {
             const qhashElm = encodeURIComponent(qhash);
             const indexElm = encodeURIComponent(index);
             const fileElm = encodeURIComponent(match.file);
-            const downloadUrl = `${API_URL}/download?job_id=${qhashElm}&ordinal=${indexElm}&file_path=${fileElm}`;
+            const downloadUrl = `/download?job_id=${qhashElm}&ordinal=${indexElm}&file_path=${fileElm}`;
```

Comment:
same nitpick as in the last one, I'm not a fan of constructing query string like that :<

Reply:
I mean, I didn't touch this line, just refactored it. This like is at least three years old, and (IIRC) not even written by me ;).

How would you refactor it? This ends up being a `href` attribute of an `<a>` (of a component of a component) element. I think I can change `<ActionDownload>` to get a `downloadAction` callback instead of `downloadUrl`, and do an API call there. But this will also remove the ability to see & copy link on hover, which I think is a bad UI (and frankly, object to - I hate websites that hide download URLs with JS buttons). So I don't really have a good idea.

Reply:
![image](https://user-images.githubusercontent.com/156560/147151725-65b6ddf5-4587-49fb-b391-a1796580a5b7.png)

Reply:
Nice! yeah, I think we can use it.